packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -96,12 +116,27 @@ module DistToFloat = {
 // Note that different logarithms don't really do anything.
 module Scale = {
   type t = scaleOperation
-  let toFn = x =>
+  let toFn = (x: t, a: float, b: float): result<float, invalidOperationError> =>
     switch x {
-    | #Multiply => \"*."
-    | #Divide => \"/."
-    | #Power => \"**"
-    | #Logarithm => (a, b) => log(a) /. log(b)
+    | #Multiply => Ok(a *. b)
+    | #Divide =>
+      if b != 0.0 {
+        Ok(a /. b)
+      } else {
+        Error(DivisionByZeroError)
+      }
+    | #Power =>
+      if a > 0.0 {
```

Comment:
I think we should allow `0.0 ** b == 0.0`

Reply:
Good spot!